src/modules/service-catalog/components/service-catalog-list-item/Search.tsx

```diff
@@ -26,12 +42,34 @@ export const Search = ({
 }: SearchProps) => {
   const { t } = useTranslation();
 
+  const SearchEndIcon = () => {
```

Comment:
We shouldn't define a component inside a component like this, but externally. It probably make sense to move the condition out of the component and just not render it when the condition is not met